templates/controllers/tab/workflow/production.tpl

```diff
@@ -11,9 +11,41 @@
 {* Help tab *}
 {help file="editorial-workflow/production.md" class="pkp_help_tab"}
 
+<script type="text/javascript">
+	// Attach the JS file tab handler.
+	$(function() {ldelim}
+		$('#submissionRevisions').pkpHandler(
+			'$.pkp.controllers.TabHandler',
+			{ldelim}
+				{assign var=versionIndex value=$latestSubmissionRevision-1}
+				selected: {$versionIndex},
+				disabled: [{$submissionRevision}]
+			{rdelim}
+		);
+	{rdelim});
+</script>
 <div id="production">
-{include file="controllers/notification/inPlaceNotification.tpl" notificationId="productionNotification" requestOptions=$productionNotificationRequestOptions}
```

Comment:
can they not be included if versioning is there -- do they not make sense any more? at least in the first step coming from copyediting they might be good to have?